src/lib/project/twitter.ex

```diff
@@ -131,6 +140,15 @@ defmodule Project.Twitter do
     |> Repo.all()
   end
 
+  @doc """
+  Gets a single favorite.
+  """
+  @spec get_favorite!(integer()) :: Favorite.t()
+  def get_favorite!(id) do
+    Favorite
+    |> Repo.get!(id)
```

Comment:
一回しかパイプしない場合は普通に書いて下さい
```suggestion
    Repo.get!(Favorite, id)
```

Reply:
ここも未対応